src/explore-education-statistics-admin/src/pages/publication/components/PublicationPublishedReleases.tsx

```diff
@@ -167,6 +197,96 @@ const PublicationPublishedReleases = ({ publicationId, releases }: Props) => {
       )}
     </>
   );
-};
+}
+
+interface PublishedReleasesTableProps {
```

Comment:
I'd split this out into it's own file as it's not a small component.

Reply:
Done